.changelog/release-v13.0.3.md

```diff
@@ -0,0 +1,151 @@
+
+		WARNING:
+			1. The Migration script 500601_party-2480.js will convert the `party` table's character-set to utf8mb4 with utf8mb4_unicode_ci collation. This cannot be undone automatically (i.e. through a rollback), but can be done manually if required.
+			2. Migration script 501002_quoteExtension-2522.js is converting `quoteExtension` 'value' column to use the following character set utf8mb4 with utf8mb4_unicode_ci collation. This cannot be undone automatically (i.e. through a rollback), but can be done manually if required.
+			3. Migration script 940101_quotePartyIdInfoExtension-2522.js is converting `quotePartyIdInfoExtension` table to use the following character set utf8mb4 with utf8mb4_unicode_ci collation. This cannot be undone automatically (i.e. through a rollback), but can be done manually if required.
+
+		The migration script will print out the current character-set and collation as per the following example:
+
+		```bash
+		❯ kubectl -n moja get po| grep centralledger-service
+		moja-centralledger-service-645dfc6674-fzpfr                      1/1     Running     0          8m49s
+		❯ kubectl -n moja logs moja-centralledger-service-645dfc6674-fzpfr -c run-migration
+
+		> @mojaloop/central-ledger@13.14.0 migrate /opt/central-ledger
+		> run-s migrate:latest seed:run
+
+
+		> @mojaloop/central-ledger@13.14.0 migrate:latest /opt/central-ledger
+		> knex $npm_package_config_knex migrate:latest
+
+		Working directory changed to /opt/central-ledger/config
+		WARNING: Migration script 500601_party-2480.js is converting PARTY table to use the following character set utf8mb4 with utf8mb4_unicode_ci collation
+		WARNING: Migration script 500601_party-2480.js - take note of the current configuration if you wish to revert= [{"character_set_name":"latin1","collation_name":"latin1_swedish_ci"}]
+		Batch 2 run: 1 migrations
+
+		> @mojaloop/central-ledger@13.14.0 seed:run /opt/central-ledger
+		> knex seed:run $npm_package_config_knex
+
+		Working directory changed to /opt/central-ledger/config
+		Ran 23 seed files
+		```
+
+## 9. Testing notes
+
+1. It is recommended that all Mojaloop deployments are verified using the [Mojaloop Testing Toolkit](https://docs.mojaloop.io/documentation/mojaloop-technical-overview/ml-testing-toolkit/). More information can be found in the [Mojaloop Deployment Guide](https://docs.mojaloop.io/documentation/deployment-guide).
+
+2. The [testing-toolkit-test-cases](https://github.com/mojaloop/testing-toolkit-test-cases/releases/tag/v13.0.0)' Golden Path collections expects:
```

Comment:
```suggestion
2. The [testing-toolkit-test-cases](https://github.com/mojaloop/testing-toolkit-test-cases/releases/tag/v13.0.2)' Golden Path collections expects:
```
until the time when we publish v13.0.3 of the ttk test cases

Reply:
oooh, we probably should update the v13.0.1/2 notes too